pageObjects/ArchivePage.js

```diff
@@ -11,12 +12,16 @@ class ArchivePage extends BasePage {
     unarchiveNote() {
         browser.actions().mouseMove(this.noteFirst).perform()
         this.iconUnarchive.click()
+        browser.wait(EC.visibilityOf(this.notificationSuccess), browser.params.customTimeout,
+            'Success notification should be visible after note unarchiving')
         this.goTo(this.optionMyNotes)
     }
 
     deleteNote() {
         browser.actions().mouseMove(this.noteFirst).perform()
         this.iconDelete.click()
+        browser.wait(EC.visibilityOf(this.notificationSuccess), browser.params.customTimeout,
```

Comment:
browser.wait(EC.presenceOf(this.notificationSuccess), browser.params.customTimeout,
            'Success notification should be visible after note deletion')